### PR TITLE
[Intel oneDNN] Enable BF16 support for binary ops: xdivy, xlogy, xlog1py

### PR DESCRIPTION
**Pull request by othakkar**
This PR adds BF16 support for the following binary ops: _xdivy_, _xlogy_, _xlog1py_

### CODE REVIEW

**Review by srvasude — APPROVED** [review]

**Comment by gbaned** [comment]
Here are the internal errors, @othakkar can you please verify ? Thank you!

Traceback (most recent call last):
  File "/build/work/46ded2af84d14c17c3b3711713f2688aa67d/google3/runfiles/google3/third_party/tensorflow/python/client/session.py", line 1378, in _do_call
    return fn(*args)
  File "/tensorflow/python/client/session.py", line 1360, in _run_fn
    self._extend_graph()
  File "/tensorflow/python/client/session.py", line 1401, in _extend_graph
    tf_session.ExtendSession(self._session)
google3.third_party.tensorflow.python.framework.errors_impl.UnknownError: <unknown>:0: error: loc(fused["Xdivy:", "Xdivy"]): 'tf.Xdivy' op operand #0 must be tensor of 128-bit complex or 64-bit complex or 16-bit float or 32-bit float or 64-bit float values, but got 'tensor<1x2x5xbf16>'
<unknown>:0: note: loc(fused["Xdivy:", "Xdivy"]): see current operation: %3 = "tf.Xdivy"(%0#0, %1#0) {device = "/device:CPU:0"} : (tensor<1x2x5xbf16>, tensor<1x2x5xbf16>) -> tensor<1x2x5xbf16>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tensorflow/python/framework/test_util.py", line 1664, in decorated
    return f(self, *args, **kwargs)
  File "/tensorflow/python/kernel_tests/math_ops/cwise_ops_binary_test.py", line 273, in testBFloat16Basic
    self._compareCpu(x, y, np.true_divide, math_ops.xdivy)
  File "/tensorflow/python/kernel_tests/math_ops/cwise_ops_binary_test.py", line 85, in _compareCpu
    tf_cpu = self.evaluate(out)
  File "/tensorflow/python/framework/test_util.py", line 2689, in evaluate
    return sess.run(tensors)
  File "/tensorflow/python/framework/test_util.py", line 2090, in run
    return super(ErrorLoggingSession, self).run(*args, **kwargs)
  File "/tensorflow/python/client/session.py", line 968, in run
    result = self._run(None, fetches, feed_dict, options_ptr,
  File "/tensorflow/python/client/session.py", line 1191, in _run
    results = self._do_run(handle, final_targets, final_fetches,
  File "/tensorflow/python/client/session.py", line 1371, in _do_run
    return self._do_call(_run_fn, feeds, fetches, targets, options,
  File "/tensorflow/python/client/session.py", line 1397, in _do_call
    raise type(e)(node_def, op, message)  # pylint: disable=no-value-for-parameter
google3.third_party.tensorflow.python.framework.errors_impl.UnknownError: Graph execution error:

<unknown>:0: error: loc(fused["Xdivy:", "Xdivy"]): 'tf.Xdivy' op operand #0 must be tensor of 128-bit complex or 64-bit complex or 16-bit float or 32-bit float or 64-bit float values, but got 'tensor<1x2x5xbf16>'
<unknown>:0: note: loc(fused["Xdivy:", "Xdivy"]): see current operation: %3 = "tf.Xdivy"(%0#0, %1#0) {device = "/device:CPU:0"} : (tensor<1x2x5xbf16>, tensor<1x2x5xbf16>) -> tensor<1x2x5xbf16>

**Comment by othakkar** [comment]
@gbaned Can you please provide me with a run command or a reproducer for the errors you posted? Thanks.

**Comment by gbaned** [comment]
@srvasude Can you please assist on above comments from @othakkar. Thank you!

**Comment by othakkar** [comment]
Hi @srvasude, can you please provide me with a run command or a reproducer for the errors posted above? Thanks.

**Comment by ashiqimranintel** [comment]
Any update on this PR?

**Comment by othakkar** [comment]
Hi @srvasude, can you please provide me with a run command or a reproducer for the errors posted above? Thanks.

File: tensorflow/python/kernel_tests/math_ops/cwise_ops_binary_test.py [hunk 1]
```diff
@@ -260,6 +264,16 @@ def testFloatVariableOverload(self):
     self.assertAllEqual(np_result, left_result)
     self.assertAllEqual(np_result, right_result)
 
+  @test_util.run_deprecated_v1
```
Comment (cantonios) [inline tensorflow/python/kernel_tests/math_ops/cwise_ops_binary_test.py:267]: Is it possible to remove this?  We want all new tests to be v2.

Comment (othakkar) [reply tensorflow/python/kernel_tests/math_ops/cwise_ops_binary_test.py]: Done!

**Review by cantonios — APPROVED** [review]

**Pull request merged**